tools/aliaslookup/aliaslookup.py

```diff
@@ -22,20 +22,21 @@ def lookup_by_aliases(client: datastore.Client,
   Returns:
     a datastore.query.Iterator
   """
-  query = client.query(kind='Bug')
-  # the "IN" operator only supports 30 identifiers as a maximum, so we need to
-  # batch into multiple a composite OR query
-  # This still caps out at 100 total filters, sadly
-  query.add_filter(
-      filter=datastore.query.PropertyFilter('aliases', 'IN', identifiers))
-  if verbose:
-    print(f'Running query {query.filters[0]} '
-          f'on {query.kind} (in {query.project})...')
-  result = list(query.fetch())
-  if verbose:
-    print(f'Retrieved {len(result)} bugs')
-
-  return result
+  for identifier in identifiers:
+    query = client.query(kind='Bug')
+    query.add_filter(
+        filter=datastore.query.PropertyFilter('aliases', 'IN', [identifier]))
```

Comment:
I think this *should* be doing a query on the [`AliasGroup`](https://github.com/google/osv.dev/blob/d51109b8198cc49cc7d47aefd29aed05902483ba/osv/models.py#L878-L881) kind, then getting the Bugs from the `bug_ids`

IIRC some gotchas: if something has 0 aliases, it won't be in any `AliasGroup`. Also, a `bug_id` in an `AliasGroup` may not exist in our db.

Reply:
> I think this _should_ be doing a query on the [`AliasGroup`](https://github.com/google/osv.dev/blob/d51109b8198cc49cc7d47aefd29aed05902483ba/osv/models.py#L878-L881) kind, then getting the Bugs from the `bug_ids`

I looked at that, then realized past me had written this, and used it for the task. I think I'll write a separate script to query `AliasGroup` for _computed_ aliases (as opposed to aliases-on-records-as-imported).